### DISCUSSION TITLE
Mobx-react old behavior (before <StrictMode/>)

### DISCUSSION THREAD

**Discussion by 1tu** [mobxjs/mobx#3794] [General] [answered]
Is it possible to make the behavior that was disabled in `mobx-react` due to `React@18 <StrictMode/>` (when props & context became reactive) configurable? I would prefer to include the old behavior (and don't use `<StrictMode/>`), since we have a lot of functionality tied to it.

**Comment by mweststrate** [comment, marked as answer]
In that case I recommend to stick to the current mobx-react version.
